content/applications/hr/payroll/reporting.rst

```diff
@@ -0,0 +1,219 @@
+=========
+Reporting
+=========
+
+The :guilabel:`Reporting` section of the *Payroll* app offers a variety of reports to choose from.
+The :guilabel:`Payroll`, :guilabel:`Meal Vouchers`, :guilabel:`Attachment of Salary`, and
+:guilabel:`DMFA` reports are universal and available for all companies regardless of location. The
+:guilabel:`Paid Time Off Allocation`, :guilabel:`273S Sheet`, and :guilabel:`274.XX Sheets` reports
+are specific to Belgian companies only.
+
+To view a report, go to :menuselection:`Payroll --> Reporting` and click on the specific report.
+
+.. image:: reporting/reports.png
+   :align: center
+   :alt: Report dashboard view.
+
+If a report is unavailable to a company, a user error pops up, stating “You must be logged in to a
+Belgian company to use this feature.”
+
+At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
+specific report. Filters show information that matches the specific filter parameters. For example,
+when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 265 days
+appear. All reports have the option to add a custom filter, and each have different default custom
+filter options. Select the parameters, then click the :guilabel:`Apply` button.
+
+.. image:: reporting/custom-filter.png
+   :align: center
+   :alt: Report dashboard view.
+
+Clicking on :guilabel:`Favorites` displays three options: :guilabel:`Save the current search`,
+:guilabel:`Add the search to the dashboard`, or :guilabel:`Insert the search to a Google
+spreadsheet`.
+
+.. image:: reporting/favorites.png
+   :align: center
+   :alt: Favorite options for reports.
+
+Payroll
+-------
+
+Clicking on :menuselection:`Payroll --> Reporting --> Payroll` displays the *Payroll Analysis*
+report.
+
+.. image:: reporting/payroll-report.png
+   :align: center
+   :alt: Report dashboard view.
+
+This shows all the payslips generated in the last 365 days. The report can display metrics for a
+variety of parameters. Click the :guilabel:`Measures` box to view a drop-down menu with the various
+options to display. The options available include: :guilabel:`# Payslip`, :guilabel:`Basic Wage`,
+:guilabel:`Basic Wage for Time Off`, :guilabel:`Days of Paid Time Off`, :guilabel:`Days of
+Unforeseen Absence`, :guilabel:`Days of Unpaid Time Off`, :guilabel:`Gross Wage`, :guilabel:`Net
+Wage`, :guilabel:`Number of Days`, :guilabel:`Number of Hours`, :guilabel:`Work Days`,
+:guilabel:`Work Hours`, and :guilabel:`Count`.
+
+.. image:: reporting/measures.png
+   :align: center
+   :alt: Report dashboard view.
+
+Line chart
```

Comment:
suggestion: for the three chart options here we can use [content tabs](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#content-tabs)

Reply:
omitting, can do later.